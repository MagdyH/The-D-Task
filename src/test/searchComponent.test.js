import React from 'react';
import renderer from 'react-test-renderer';
import Search from '../components/searchComponent';
import configureStore from 'redux-mock-store';

const initialState = {}
const mockStore = configureStore();
const store = mockStore(initialState);

describe('Search component', () => {
    it('should render snapshot', () => {
      const component = renderer.create(<Search store={store} />);
  
      const tree = component.toJSON();
      expect(tree).toMatchSnapshot();
    });

});