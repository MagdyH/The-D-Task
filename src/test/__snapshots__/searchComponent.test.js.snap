// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Search component should render snapshot 1`] = `
<div
  className="mt-4"
/>
`;
